src/Orient/Orient.Client/Protocol/DatabasePool.cs

```diff
@@ -52,6 +57,13 @@ internal DatabasePool(string hostname, int port, string databaseName, ODatabaseT
             }
         }
 
+        internal void ReEstablishConnections()

```

Comment:
`DropAndEstabilishAllConnections` would be a more informative name for this method.